plaso/engine/artifact_filters.py

```diff
@@ -118,17 +117,24 @@ def BuildFindSpecs(self, environment_variables=None):
               key_value['key'] for key_value in source.key_value_pairs]):
             if self.CheckKeyCompatibility(key_path):
               find_specs = self.BuildFindSpecsFromRegistryArtifact(key_path)
-              find_specs_per_source_type[
+              self.find_specs_per_source_type[
```

Comment:
My advise would be to have 1 statement per line

```
artifact_group = self.find_specs_per_source_type[artifact_types.TYPE_INDICATOR_WINDOWS_REGISTRY_KEY]
artifact_group.extend(find_specs)
```

this make debugging easier